### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by kessio**
**Hello 👋🏾** 

**The following has been implemented on this pull request project;**
- [x] A login page.
- [x] A registration page.
- [x] A food list (with all CRUD implementation, except for 'update').
- [x] A recipe list (with all CRUD implementation, except for 'update').
- [x] A general shopping list view (all missing food for all your recipes and total price).
- [x] A list of all public recipes from other users with their names and total prices.
- [x] All tests have passed

This is a [video](https://www.loom.com/share/8310ac3706254696bfef6fe584c74812) demonstration of the project as well

### CODE REVIEW

File: app/controllers/recipe_foods_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class RecipeFoodsController < ApplicationController
```
Comment (hiromataba) [inline app/controllers/recipe_foods_controller.rb:1]: - [ ] Please check my comment in the recording , 

 
https://user-images.githubusercontent.com/75126481/219119527-90ab9fc7-e77a-414d-af8c-51307c19c900.mov


## This prevents me from seeing other functionalities so for the next reviewer will check the other functionalities and may require changes , Please double check before you request for another review thanks 😉

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @kessio and @Jaymelfah    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file 🎉 
- Your test are passing 🚀 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/recipe_foods_controller.rb (hunk 1 above)
Comment (Jaymelfah) [reply app/controllers/recipe_foods_controller.rb:1]: Thanks for the review. We have double checked on both our local devices and it works. This is a link to show how it works as well. [https://www.loom.com/share/8310ac3706254696bfef6fe584c74812](project)

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi team, 

To highlight 👍🏽 
✔️ App runs smoothly
✔️ Design matches wireframe

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
## STATUS: APPROVED ♻️♻️
Hi <STUDENT_MENTION>, 

To highlight 👍🏽 
✔️ 
✔️ 
✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._











Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**